pkgs/top-level/all-packages.nix

```
@@ -326,6 +326,8 @@ in
 
   fetchhg = callPackage ../build-support/fetchhg { };
 
+  fetchfirefoxaddon = callPackage ../build-support/fetchfirefoxaddon {};
```

Also many old fetchers deviate from it, it would be nice
to use camelCase because this is what we use for many
other nix function/builders as well:

```suggestion
  fetchFirefoxAddon = callPackage ../build-support/fetchfirefoxaddon {};
```

This makes it easier to distinguish between packages and builder functions.